### PR TITLE
Ll firepower rest api105 json

### PR DESCRIPTION
**Pull request by ReneeBornstein**
created titles for table of contents

### CODE REVIEW

File: labs/firepower-restapi-104/firepower-restapi-104.json [hunk 1]
```diff
@@ -30,9 +30,11 @@
         "title": "Cisco Security REST API"
     }],
     "files": [{
-        "title": "1.md"
+       "title": "FMC - REST - API - 104",
```
Comment (annegentle) [inline labs/firepower-restapi-104/firepower-restapi-104.json]: Don't need these dashes. Also I believe there could be spelling errors in this file, can you check?

File: labs/firepower-restapi-105/firepower-restapi-105.json [hunk 2]
```diff
@@ -30,9 +30,11 @@
         "title": "Cisco Security REST API"
     }],
     "files": [{
-        "title": "1.md"
+        "file": "1.md",
+        "title": "FMC - REST - API - 105"
```
Comment (annegentle) [inline labs/firepower-restapi-105/firepower-restapi-105.json]: Same here, don't need dashes. FMC REST API 105 is sufficient.

**Review by annegentle — CHANGES_REQUESTED** [review]
I've updated the first three labs JSON files I believe in the master branch, so you can change 104 and 105.

**Comment by ReneeBornstein** [comment]
Okay, I wasn’t sure, and was being consistent. I will change them, and do all of the others properly.

BTW, the reason I’ve only done a few today is that I was also trying to redo the document explaining the tasks for editing each kind of doc. For the JSON doc, it takes 17 steps to get done properly. I’ve attached that section to show you what I’ve written, and make sure you’re okay with it.

**Review by annegentle — APPROVED** [review]

**Pull request merged**